bot/seasons/christmas/adventofcode.py

```diff
@@ -130,23 +130,35 @@ def __init__(self, bot: commands.Bot):
 
         await ctx.invoke(self.bot.get_command("help"), "adventofcode")
 
-    @adventofcode_group.command(name="notifications", aliases=("notify", "notifs"), brief="Notifications for new days")
-    async def aoc_notifications(self, ctx: commands.Context):
+    @adventofcode_group.command(name="subscribe",
+                                aliases=("sub", "notifications", "notify", "notifs"),
+                                brief="Notifications for new days")
+    async def aoc_subscribe(self, ctx: commands.Context):
         """
         Assign the role for notifications about new days being ready.
+        """
+        role = ctx.guild.get_role(AocConfig.role_id)
 
-        Call the same command again to end notifications and remove the role.
+        if role not in ctx.author.roles:
+            await ctx.author.add_roles(role)
+            await ctx.send("Okay! You have been __subscribed__ to notifications about new Advent of Code tasks. "
+                           f"You can run `{ctx.prefix}unsubscribe` to disable them again for you.")
+        else:
+            await ctx.send("Hey, you already are receiving notifications about new Advent of Code tasks. "
+                           f"If you don't want them any more, run `{ctx.prefix}unsubscribe` instead.")
+
+    @adventofcode_group.command(name="unsubscribe", aliases=("unsub"), brief="Notifications for new days")
```

Comment:
```suggestion
    @adventofcode_group.command(name="unsubscribe", aliases=("unsub",), brief="Notifications for new days")
```
Needs trailing comma for valid tuple.